mne/io/eeglab/eeglab.py

```diff
@@ -60,8 +59,7 @@ def _to_loc(ll):
 
 
 def _get_info(eeg, montage, eog=()):
```

Comment:
why does this function both get info and set info? shouldn't those be separate functions? also, don't we have similar functions elsewhere?